bucketlist/src/main/kotlin/io/aexp/bucketlist/BucketListClient.kt

```diff
@@ -34,6 +35,17 @@ interface BucketListClient {
             Observable<PagedResponse<PullRequestCommit>>
 
     /**
+     * @param contextLines "the number of context lines to include around added/removed lines in the diff"
+     * @param whitespace "optional whitespace flag which can be set to 'ignore-all'" Default value is "show"
+     * @param withComments "true to embed comments in the diff (the default); otherwise, false to stream the diff without comments"
```

Comment:
consider a `CommentMode` enum or something to make a method call more readable: `foo.getPrDiff(..., true)` vs `foo.getPrDiff(..., WithComments)`. WDYT?